src/components/Table/GridTable.stories.tsx

```diff
@@ -199,6 +203,323 @@ export function NestedRows() {
   );
 }
 
+export function NestedCards() {
+  const nameColumn: GridColumn<NestedRow> = {
+    header: () => "Name",
+    parent: (row) => ({
+      content: <div css={Css.base.$}>{row.name}</div>,
+      value: row.name,
+    }),
+    child: (row) => ({
+      content: <div css={Css.sm.$}>{row.name}</div>,
+      value: row.name,
+    }),
+    grandChild: (row) => ({
+      content: <div css={Css.xs.$}>{row.name}</div>,
+      value: row.name,
+    }),
+  };
+  const spacing = { brPx: 4, pxPx: 8, spacerPx: 8 };
+  const nestedStyle: GridStyle = {
+    nestedCards: {
+      parent: { bgColor: Palette.Gray500, ...spacing },
+      child: { bgColor: Palette.Gray200, bColor: Palette.Gray600, ...spacing },
+      grandChild: { bgColor: Palette.Green200, bColor: Palette.Green400, ...spacing },
+    },
+  };
+
+  return (
+    <GridTable
+      columns={[nameColumn]}
+      {...{ rows }}
+      style={nestedStyle}
+      sorting={{ on: "client", initial: [0, "ASC"] }}
+    />
+  );
+}
+
+// Every row by-definition opens or closes a card.
+// Every row has a space between.
+
+export function NestedCardsProofOfConcept() {
```

Comment:
Will delete this but this is what I was using to prototype the DOM structure by hand before automating.